### DISCUSSION TITLE
@reduxjs/toolkit not working in android standalone

### DISCUSSION THREAD

**Discussion by Jheysoon** [callstack/repack#522] [Q&A] [answered]
![image](https://github.com/callstack/repack/assets/8098903/8d62eb6e-6e89-4e7f-9fde-bd7856d768c0)

When running in android standalone this will show.

Adding alias does not work

![image](https://github.com/callstack/repack/assets/8098903/3ab04534-b945-449d-a572-9f2b3c7dce4b)

![image](https://github.com/callstack/repack/assets/8098903/02e45f7f-37d3-476b-9c1a-610d00d250b5)

Already followed instructions from https://github.com/callstack/repack/issues/359

**Comment by jbroma** [comment, marked as answer]
hey @Jheysoon,

please refer to this issue: https://github.com/callstack/repack/issues/500 (especially [this](https://github.com/callstack/repack/issues/500#issuecomment-1991677970))

**Reply by Jheysoon** [reply]
Thanks @jbroma that works..

But currently having a problem with fetch API.

My code for fetch does not work for some reason.

Do you have any idea for that ?

**Reply by Jheysoon** [reply]
Also tried axios but still issue my async await code does not work

**Reply by jbroma** [reply]
are you getting any specific errors?

**Reply by Jheysoon** [reply]
![image](https://github.com/callstack/repack/assets/8098903/b4666455-05ad-4d47-b304-2c96bd3180d2)

It just got stuck. It does not console **b here ##**

**Reply by Jheysoon** [reply]
Also i think this is a Android only.

I also did not encountered the redux/toolkit issue in iOS.

Also fetch is not a issue in iOS

**Reply by jbroma** [reply]
I'm not sure what might be happening there, please open an issue with reproduction repository so we can dig deeper into this.

**Reply by Jheysoon** [reply]
No worries @jbroma . I did make it work by copy-pasting the host app and re-create the mini app from there.

Since i notice that the host app fetch API is working fine.

Anyways thanks for the help again
